### DISCUSSION TITLE
Does unredir-if-possible=true disable vsync?

### DISCUSSION THREAD

**Discussion by ByteDrummer** [yshui/picom#963] [Q&A] [answered]
I just want to confirm. Does setting `unredir-if-possible=true` disable vsync while there is a fullscreen window?

**Comment by yshui** [comment, marked as answer]
it disables the compositor, whether you have vsync or not after that is not in the control of the compositor.

**Reply by ByteDrummer** [reply]
Sorry, I meant specifically the vsync done by picom since I have `vsync = true` in my config.

**Reply by yshui** [reply]
Right, everything the compositor does would be disabled, including vsync.
